File: 4th week math char string/PythagoreanAngles.c
#include <stdio.h>
#include <math.h>

int main(){
	
	double ab = 4;
	double ac = 3;
	double bc = sqrt(ab*ab+ac*ac);
	
	/* compute radian */
	double a = (acos((ac*ac-ab*ab-bc*bc)/(-2*bc*ab)));
	double b = (acos((ab*ab-bc*bc-ac*ac)/(-2*bc*ac)));
	
	/* compute angle */
	double A = a*180.0/M_PI;
	double B = b*180.0/M_PI;
	
	/* display results */
	printf("alpha = %.2lf radians, %.1lf degrees\n", a, A);
	printf("beta = %.2lf radians, %.1lf degrees", b, B);
	
	return 0;
}

File: 4th week math char string/PythagoreanAngles.java

public class PythagoreanAngles{
	public static void main(String[] args){
		
		// lenght of sides
		double ab = 4;
		double ac = 3;
		double bc = Math.sqrt(ab*ab + ac*ac);
				
		// compute radian
		double aradian = Math.acos((ac*ac-ab*ab-bc*bc)/(-2*bc*ab));
		double bradian = Math.acos((ab*ab-bc*bc-ac*ac)/(-2*bc*ac));
		
		// compute degree
		double adegree = Math.round((Math.toDegrees(aradian))*10)/10.0;
		double bdegree = Math.round((Math.toDegrees(bradian))*10)/10.0;
		// display
		
		System.out.println("alpha = " + Math.rint(aradian*100)/100.0 + ", " + adegree + " degrees");
		System.out.println("beta = " + Math.rint(bradian*100)/100.0 + ", " + bdegree + " degrees");		
	}
}